definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -7047,5 +7047,12 @@
     "CaseFieldID": "hearingOrderOtherDocuments",
     "UserRole": "caseworker-divorce-financialremedy-judiciary",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "finalOrderCollection",
+    "UserRole": "[APPSOLICITOR]",
```

Comment:
if both applicant and respondent solicitor have authorisation can we just use the IDAM role here rather than the case role?